soc/CMakeLists.txt

```diff
@@ -16,6 +16,11 @@ if(HWMv1)
     add_subdirectory(${SOC_DIR}/${ARCH}/${SOC_PATH} soc/${ARCH}/${SOC_PATH})
   endif()
 elseif(HWMv2)
+  # Some soc (riscv arch) might have a common dir
+  if(EXISTS ${ZEPHYR_BASE}/soc/v2/${ARCH}/CMakeLists.txt)
```

Comment:
riscv socs share the same `common` folder

Reply:
this is going to be dropped soon.